spec/cc/analyzer/path_entries_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "spec_helper"
+
+module CC::Analyzer
+  describe PathPatterns do
+    include FileSystemHelpers
+
+    around do |test|
+      within_temp_dir { test.call }
+    end
+
+    describe "#entries" do
+      it "filters out trailing slashes" do
+        make_file("lib/foo.rb")
+
+        paths = PathEntries.new("lib/").entries
+
+        paths.sort.must_equal(["lib", "lib/foo.rb"])
+      end
```

Comment:
The class doesn't accept multiple paths, it just generates entries for a given path. I'll add a test for no slash though.